make/sandbox-ui.mk

```diff
@@ -0,0 +1,118 @@
+SANDBOX_UI_NS := sandbox-ui
+SANDBOX_PLUGIN_IMAGE_NAME := sandbox-rhdh-plugin
+TAG := latest
+PLATFORM = linux/amd64
+RHDH_PLUGINS_DIR ?= $(TMPDIR)rhdh-plugins
+AUTH_FILE := /tmp/auth.json
+IMAGE_TO_PUSH_IN_QUAY ?= quay.io/$(QUAY_NAMESPACE)/sandbox-rhdh-plugin:$(TAG)
+OPENID_SECRET_NAME=openid-sandbox-public-client-secret
+PUSH_SANDBOX_IMAGE ?= true
+
+.PHONY: deploy-sandbox-ui
+deploy-sandbox-ui: REGISTRATION_SERVICE_API=https://$(shell oc get route registration-service -n ${HOST_NS} -o custom-columns=":spec.host" | tr -d '\n')/api/v1
+deploy-sandbox-ui: HOST_OPERATOR_API=https://$(shell oc get route api -n ${HOST_NS} -o custom-columns=":spec.host" | tr -d '\n')
+deploy-sandbox-ui: RHDH=https://rhdh-${SANDBOX_UI_NS}.$(shell oc get ingress.config.openshift.io/cluster -o jsonpath='{.spec.domain}')
+deploy-sandbox-ui: check-registry
+deploy-sandbox-ui:
+	@echo "sandbox ui will be deployed in '${SANDBOX_UI_NS}' namespace"
+	$(MAKE) create-namespace SANDBOX_UI_NS=${SANDBOX_UI_NS}
+ifeq ($(PUSH_SANDBOX_IMAGE),true)
+	$(MAKE) push-sandbox-plugin
+endif
+	$(MAKE) create-pull-secret 
+	kustomize build deploy/sandbox-ui/e2e-tests | REGISTRATION_SERVICE_API=${REGISTRATION_SERVICE_API} \
+			HOST_NS=${HOST_NS} \
+			HOST_OPERATOR_API=${HOST_OPERATOR_API} \
+			SANDBOX_UI_NS=${SANDBOX_UI_NS} \
+			SANDBOX_PLUGIN_IMAGE=${IMAGE_TO_PUSH_IN_QUAY} \
+			RHDH=${RHDH} envsubst | oc apply -f -
+	$(MAKE) configure-oauth-idp
+	@echo "restarting registration-service to apply toolchainconfig changes"
+	@oc -n ${HOST_NS} rollout restart deploy/registration-service
+	@oc -n ${SANDBOX_UI_NS} rollout status deploy/rhdh
+	@echo "Developer Sandbox UI running at ${RHDH}"
+
+create-pull-secret: OS_IMAGE_REGISTRY=$(shell oc get route default-route -n openshift-image-registry --template='{{ .spec.host }}' 2>/dev/null || true)
+create-pull-secret:
+	@oc create secret docker-registry pull-secret \
+		--docker-server=${OS_IMAGE_REGISTRY} \
+		--docker-username=${OC_WHOAMI} \
+		--docker-password=${OC_WHOAMI_TOKEN} \
+		--namespace=${SANDBOX_UI_NS}
+	@oc extract secret/pull-secret -n ${SANDBOX_UI_NS} --keys=.dockerconfigjson --to=- > ${AUTH_FILE}
+	@oc create secret generic rhdh-dynamic-plugins-registry-auth \
+		--from-file=auth.json=${AUTH_FILE} \
+		--namespace=${SANDBOX_UI_NS}
+	rm ${AUTH_FILE}
```

Comment:
this part as well - do we still require this or it can be removed now that we use quay.io ?